CHANGES.md

```diff
@@ -1,5 +1,10 @@
 # Release Notes
 
+## 0.1.9
+
+This release fixes a bug wherein, when using the -o/--output option Lambdex would fail to write the
```

Comment:
This description can't be true. The chmod is after the copy, so the writing definetely work or the fix doesn't fix if I read all this correctly: https://github.com/pantsbuild/lambdex/pull/30/files#diff-4ff255eb08b5673f8ce58b8260fe52c4004b235e6cbef0f954d02b752f934a47R63-R64

https://github.com/pantsbuild/lambdex/blob/c1b3dc3c0aa73477595f56d2d4ecec5be19cde21/lambdex/bin/lambdex.py#L61-L64

Reply:
Ah, right, the copy is not the only write.